jest/setupTests.ts

```diff
@@ -1,5 +1 @@
-import Enzyme from 'enzyme';
-import Adapter from 'enzyme-adapter-react-16';
-import 'jest-styled-components';
-
-Enzyme.configure({ adapter: new Adapter() });
+import '@testing-library/jest-dom/extend-expect';
```

Comment:
Please use prettier (newline should be added)

Reply:
oh, my perfect environment... fixed